tools/Customization/DevHome.Customization/Models/SourceControlIntegration.cs

```diff
@@ -35,7 +37,7 @@ public static bool ValidateSourceControlExtension(string extensionCLSID, string
             {
                 Log.Error("Could not open local repository.");
                 Log.Error(result.Result.DisplayMessage);
-                return false;
+                return new SourceControlValidationResult(ResultType.Failure, ErrorType.OpenRepositoryFailed, null);
```

Comment:
Should pass along the error/result information from the `GetLocalRepositoryResult` so we can display that information to the user.